bali/events/handler.py

```diff
@@ -79,7 +79,7 @@ def handle():
                 queues=[items[0].queue],
                 accept=['json'],
                 callbacks=[i.callback for i in items],
-                auto_declare=False
+                auto_declare=True
```

Comment:
Can you increase the coverage of unit tests？

Reply:
resolved